docsite/index.html

```diff
@@ -1319,311 +1319,350 @@ <h5>Modify Opacity Example</h5>
           </code></pre>
         </div>
       </div>
+
+      <div class="copy-options">
+        <form>
+          <p>Copy type</p>
+          <div class="options">
+            <div class="input-group">
+              <input type="radio" name="copy-format" id="prop" checked>
+              <label for="prop">
+                CSS Property <span class="example">Ex: var(--gray-1)</span>
+              </label>
+            </div>
+            <div class="input-group">
+              <input type="radio" name="copy-format" id="color-code">
+              <label for="color-code">
+                Color Code <span class="example">Ex: rgb(3, 5, 7)</span>
+              </label>
+            </div>
+          </div>
+        </form>
+
+        <form>
+          <p>Copy format</p>
+          <div class="options">
```

Comment:
i dont think this needs a refactor, this is more of a comment: 
I personally would have used a <fieldset> and <legend> for these 2 <forms>, which then would make listening for user inputs only 1 listener on each fieldset and the added bonus you can disable a fieldset which disables the children. slightly simpler to manage i think, but would need some css updates to remove the styling of the html elements.

all in all, what you have here is accessible to keyboard and screen reader, so no refactor needed 👍🏻